clients/imodels-client-management/src/base/interfaces/apiEntities/ChangesetInterfaces.ts

```diff
@@ -21,52 +25,97 @@ export enum ContainingChanges {
   GlobalProperties  = 1 << 5
 }
 
+/** Application information. */
 export interface Application {
+  /** Application name. */
   name: string;
 }
 
+/** Synchronization information. */
 export interface SynchronizationInfo {
+  /** List of files that were processed by the synchronization. */
   changedFiles: string[];
 }
 
+/** DTO for links that belong to Changeset entity returned from iModels API. */
 export interface ChangesetLinks {
+  /** Link to upload the Changeset file. Points to file storage. */
   upload: Link;
+  /** Link to download the Changeset file. Points to file storage. */
   download: Link;
+  /**
+   * Link to confirm the Changeset file upload and complete the creation process. Points to a specific
+   * Changeset in iModels API.
+   */
   complete: Link;
+  /** Link to a Named Version created on the Changeset. Points to a specific Named Version in iModels API. */
   namedVersion?: Link;
+  /**
+   * Link to a Checkpoint that is created on a current or preceding Changeset. Points to a specific Checkpoint
+   * in iModels API.
+   * */
   currentOrPrecedingCheckpoint?: Link;
 }
 
+/** Minimal representation of a Changeset. */
 export interface MinimalChangeset {
+  /** Changeset id. */
   id: string;
+  /** Changeset display name. */
   displayName: string;
+  /** Changeset description. */
   description: string;
+  /** Changeset index. */
   index: number;
+  /** Changeset parent id. Equals to empty string if the Changeset is first in sequence. */
```

Comment:
Id of the parent Changeset

Reply:
Fixed: https://github.com/iTwin/imodels-clients/blob/1aac582dbc6b1ead914650a280cabe53a4f0487e/clients/imodels-client-management/src/base/interfaces/apiEntities/ChangesetInterfaces.ts#L70